images/python/bootstrap.py

```diff
@@ -234,12 +229,17 @@ def filter(self, record):
     stopWithError("Function name is not defined")
 
 try:
-    with file(payloadFileName) as f:
+    if payloadFileName:
+        stopWithError("No payload present")
```

Comment:
`if not os.path.isfile(payloadFileName):
    stopWithError("No payload present")`
?